canvas_modules/common-canvas/src/common-canvas/svg-canvas-d3.js

```diff
@@ -114,9 +129,7 @@ export default class SVGCanvasD3 {
 				// will interfere with drawing of the canvas as the result of any
 				// keyboard action.
 				this.canvasController.closeTip();
-
 				const actions = this.canvasController.getKeyboardConfig().actions;
-
```

Comment:
I guess it doesn't matter too much but please avoid removing blank lines (like those above) unless you think it greatly improves the code for some reason.